index.html

```diff
@@ -0,0 +1,314 @@
+        <div
+          class="h-52 w-52 text-white rounded-2xl shadow-xl flex justify-center align-items-center cursor-pointer home__page__brand__logo bg-slate-900"
+        >
+          <h3 class="text-2xl text-white shadown-2xl p-2 w-full font-semibold">
+            pens & markers
+          </h3>
+        </div>
+        <div
+          class="h-52 w-52 text-white rounded-2xl shadow-xl flex justify-center align-items-center cursor-pointer home__page__brand__logo bg-slate-900"
+        >
+          <h3 class="text-2xl text-white shadown-2xl p-2 w-full font-semibold">
+            paints & watercolors
+          </h3>
+        </div>
+        <div
+          class="h-52 w-52 text-white rounded-2xl shadow-xl flex justify-center align-items-center cursor-pointer home__page__brand__logo bg-slate-900"
+        >
+          <h3 class="text-2xl text-white shadown-2xl p-2 w-full font-semibold">
+            crayons & color pencils
+          </h3>
+        </div>
+        <div
+          class="h-52 w-52 text-white rounded-2xl shadow-xl flex justify-center align-items-center cursor-pointer home__page__brand__logo bg-slate-900"
+        >
+          <h3 class="text-2xl text-white shadown-2xl p-2 w-full font-semibold">
+            notebooks & notepads
+          </h3>
+        </div>
+      </div>
+      <!-- featured categories -->
+
+      <div class="spacer-3rem"></div>
+      <div class="spacer-3rem"></div>
+      <div class="spacer-3rem"></div>
+      <div class="spacer-3rem"></div>
+
+      <!-- video ad section -->
+      <div class="hompage__iframe">
+        <iframe
+          class="w-full h-96"
+          src="https://www.youtube.com/embed/zOnWsfucES0"
+          title="YouTube video player"
+          frameborder="0"
+          allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+          allowfullscreen
+        ></iframe>
+      </div>
+      <!-- video ad section -->
+      <div class="spacer-3rem"></div>
+      <div class="spacer-3rem"></div>
+      <div class="spacer-3rem"></div>
+      <div class="spacer-3rem"></div>
+    </main>
+    <!-- main -->
+
+    <!-- footer -->
+    <footer
+      class="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-rose-200"
+      style="bottom: 0"
+    >
+      <div class="text-grey-700">
+        Made with
+        <span class="text-rose-700 font-bold ml-1 mr-1">&lt;/&gt;</span>by
+
+        <a class="ml-1 text-hover-rose-600">Ananya</a>
+      </div>
+      <div class="flex mt-3 align-items-center justify-center">
+        <a href="https://github.com/Ananyamadhu08" target="_blank">
+          <i class="fab fa-github mr-2 text-rose-700"></i>
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-rose-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
+        >
```

Comment:
Suggestion : It is good to use rel="noopener noreferrer" with target="_black" to improve security.

Reply:
Okay will add the changes